tests/rs-int2hex.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# add 2018-04-04 by Pascal Withopf, released under ASL 2.0
+. $srcdir/diag.sh init
+. $srcdir/diag.sh generate-conf
+. $srcdir/diag.sh add-conf '
+module(load="../plugins/imtcp/.libs/imtcp")
+input(type="imtcp" port="13514")
+
+set $!var = int2hex(61);
+set $!var2 = int2hex(-13);
+
+template(name="outfmt" type="string" string="-%$!var%- -%$!var2%-\n")
+:syslogtag, contains, "tag" action(type="omfile" template="outfmt"
+			         file="rsyslog.out.log")
+'
+. $srcdir/diag.sh startup
+. $srcdir/diag.sh tcpflood -m1 -M "\"<129>Mar 10 01:00:00 172.20.245.8 tag: msgnum:1\""
+. $srcdir/diag.sh shutdown-when-empty
+. $srcdir/diag.sh wait-shutdown
+echo '-3d- -fffffffffffffff3-' | cmp rsyslog.out.log
```

Comment:
cmp needs a dash as first argument, else the test will fail on someplatforms, e.g. BSD